packages/lib/sanitizeCalendarObject.ts

```diff
@@ -0,0 +1,12 @@
+import { DAVObject } from "tsdav";
+
+export const sanitizeCalendarObject = (obj: DAVObject) => {
+  return obj.data
+    .replaceAll("\r\n", "\r")
+    .replaceAll("\r", "\r\n")
```

Comment:
The reason we're going from `\r\n` to `\r` and then back from `\r` to `\r\n` here is because some of the CalDAV services return the EOL as `\r` while others as `\r\n` and the `ICAL.parse` expects it to have `\r\n` as EOL, so this just ensures all the returned Calendar objects have `\r\n` as EOL.

Reply:
Just a nitpick `replaceAll` is not supported on browsers. And it is generally deprecated.